test/lib.fetching.content-services.facebook-content-service.test.js

```diff
@@ -72,35 +72,29 @@ describe('Facebook content service', function() {
 
   it('should get new comments from old posts', function(done) {
 
-    // Fetch first batch
-    var service = stubWithFixture('facebook-1.json');
-    service.fetch({maxCount: 50}, function(){});
-
-    // Stub for second batch
-    stubWithFixture('facebook-2.json', service);
-
+    var service = stubWithFixture('facebook-2.json');
```

Comment:
Right, but I think the reason fixture 1 was loaded first was to test the duplication avoidance stuff I described above.